src/components/ui/multi-select.tsx

```
@@ -35,201 +39,208 @@ interface MultiSelectProps
-            )}
+    <div
+      className="flex flex-col h-full overflow-hidden"
+      onKeyDown={(e) => {
+        if (e.key === "Enter") {
+          e.preventDefault();
+          e.stopPropagation();
+          onValueChange(selectedValues);
+          setOpen(false);
+        }
+      }}
+    >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)